build-system/compile/compile.js

```diff
@@ -358,8 +348,6 @@ function compile(
       delete compilerOptions.define;
     }
 
-    compilerOptions.js_module_root.push(SRC_TEMP_DIR);
```

Comment:
just wanted to confirm we don't need this shadowing anymore?

Reply:
Correct. There is no temporary directory any more. All transforms are done directly on the Vinyl stream. Nothing is written to disk until after closure does its thing.